docs/contributor/04-local-test-setup.md

```diff
@@ -16,283 +13,161 @@ This setup is deployed with the following security features enabled:
 * Strict mTLS connection between Kyma Control Plane (KCP) and SKR clusters
 * SAN Pinning (SAN of client TLS certificate needs to match the DNS annotation of a corresponding Kyma CR)
 
-## Procedure
+## Prerequisites
 
-### KCP Cluster Setup
+The following tooling is required in the versions defined in [`versions.yaml`](../../versions.yaml):
 
-1. Create a local KCP cluster:
+- docker
+- go
+- golangci-lint
+- istioctl
+- k3d
+- kubectl
+- kustomize
+- [modulectl](https://github.com/kyma-project/modulectl)
+- yq
 
-    ```shell
-    k3d cluster create kcp-local --port 9443:443@loadbalancer \
-    --registry-create k3d-registry.localhost:0.0.0.0:5111 \
-    --k3s-arg '--disable=traefik@server:0' \
-    --k3s-arg --tls-san=host.k3d.internal@server:* 
-    ```
+## Procedure
 
-2. Open `/etc/hosts` file on your local system:
+Execute the following scripts from the project root.
 
-   ```shell
-   sudo nano /etc/hosts
-   ```
+### 1. Create Test Clusters
 
-   Add an entry for your local k3d registry created in step 1:
+Create local test clusters for SKR and KCP.
 
-   ```txt
-   127.0.0.1 k3d-registry.localhost
-   ```
+```sh
+K8S_VERSION=$(yq e '.k8s' ./versions.yaml)
+CERT_MANAGER_VERSION=$(yq e '.certManager' ./versions.yaml)
+./scripts/tests/create_test_clusters.sh --k8s-version $K8S_VERSION --cert-manager-version $CERT_MANAGER_VERSION
+```
 
-3. Install the following prerequisites required by Lifecycle Manager:
+### 2. Install the CRDs
 
-   1. Istio CRDs using `istioctl`:
+Install the CRDs to the KCP cluster.
 
-      ```shell
-      brew install istioctl && \
-      istioctl install --set profile=demo -y
-      ```
+```sh
+./scripts/tests/install_crds.sh
+```
 
-   2. `cert-manager` by Jetstack:
+### 3. Deploy lifecycle-manager
 
-       ```shell
-       kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.13.3/cert-manager.yaml
-       ```
+#### 3.1 Deploy lifecycle-manager from a Registry
```

Comment:
```suggestion
You can deploy Lifecycle Manager either from the registry or local sources. Choose one of the below options:

3.1 Deploy a built image from the registry, for example, the `latest` image from the `prod` registry.
```
It could look cleaner if we don't use H4 but just an ordered list. Then the title case is not necessary and we can write the steps straight away.